src/ch/ehi/ili2db/fromili/FromIliRecordConverter.java

```diff
@@ -709,11 +710,17 @@ public void generateAttr(DbTable dbTable,Viewable aclass,AttributeDef attr,Integ
 					dbColExts.add(ret);
 				}
 			}
-		}else if (type instanceof PolylineType){
-			String attrName=attr.getContainer().getScopedName(null)+"."+attr.getName();
-			DbColGeometry ret = generatePolylineType((PolylineType)type, attrName);
-			setCrs(ret,epsgCode);
-			dbCol.value=ret;
+		}else if (type instanceof PolylineType) {
+			String attrName = attr.getContainer().getScopedName(null) + "." + attr.getName();
+			DbColGeometry ret = generatePolylineType((PolylineType) type, attrName);
+			setCrs(ret, epsgCode);
+			dbCol.value = ret;
```

Comment:
Das rückgängig machen der whitespace changes ist nicht ganz geglückt, um gleich zeichen etc. hatte es im original zum teil keine Abstände.

Reply:
Ich würde den change auch hier mit reinnehmen. Verbesserung der Leserlichkeit in kleinen Schritten.